docs/setup/install-new.md

```diff
@@ -27,14 +27,16 @@ Follow the steps below to create an Azure Active Directory (AAD) [service princi
 
 1. Create AD service principal ([Read more about RBAC](https://docs.microsoft.com/en-us/azure/role-based-access-control/overview)):
     ```bash
-    az ad sp create-for-rbac --skip-assignment
+    az ad sp create-for-rbac --skip-assignment -o json > auth.json
+    appId=$(jq -r ".appId" auth.json)
```

Comment:
Since we introduce `jq` as a dependency, we may want to call that out for folks who are not using Cloud Shell.